docs/services/blockstorage.rst

```diff
@@ -1,89 +1,187 @@
+    81c7a7e5-01a5-44bb-9b43-0cc9f7c4e423		     75	   available		            SATA				[]															<nil>
+    a7c97db7-a0d3-495b-9ba3-1cb9dd9cf12c	Store	 75	   in-use			            	SATA			[map[host_name:<nil> device:/dev/xvdb server_id:8a254ea3-77b5-4f74-a893-8d2d51ae2cca id:a7c97db7-a0d3-495b-9ba3-1cb9dd9cf12c volume_id:a7c97db7-a0d3-495b-9ba3-1cb9dd9cf12c]]	<nil>
+    ca50fdfd-21f2-47e9-8ede-1518cb7467af	Store  150	 in-use				            SSD				[map[host_name:<nil> device:/dev/xvda server_id:8a254ea3-77b5-4f74-a893-8d2d51ae2cca id:ca50fdfd-21f2-47e9-8ede-1518cb7467af volume_id:ca50fdfd-21f2-47e9-8ede-1518cb7467af]]	<nil>
 
 ``create``
 ^^^^^^^^^^
-Usage::
+Creates a volume::
 
     rack block-storage volume create --size <volumeSize> [optional flags]
+    
+**Response**
+
+.. code::
+    
+    $ rack block-storage volume create --size 75 --name Response
+    ID		66dcbe53-1b62-4a15-adc2-e46e78b95f8b
+    Name		Response
+    Description
+    Size		75
+    VolumeType	SATA
+    SnapshotID
+    Attachments	[]
+    CreatedAt	2015-08-05T19:48:28.000000
 
 ``get``
 ^^^^^^^
-Usage::
+Retrieves details on a specified volume::
 
     rack block-storage volume get --id <volumeID> [optional flags]
     rack block-storage volume get --name <volumeName> [optional flags]
     (echo volumeID1 && echo volumeID2) | rack block-storage volume get --stdin id [optional flags]
 
+**Response**
+
+.. code::
+    
+    $ rack block-storage volume get --name Response
+    ID		66dcbe53-1b62-4a15-adc2-e46e78b95f8b
+    Name		Response
+    Description
+    Size		75
+    VolumeType	SATA
+    SnapshotID
+    Attachments	[]
+    CreatedAt	2015-08-05T19:48:28.000000
+
+
 ``update``
 ^^^^^^^^^^
-Usage::
+Updates the name and description of a volume::
 
     rack block-storage volume update --id <volumeID> [optional flags]
     rack block-storage volume update --name <volumeName> [optional flags]
 
+
 ``delete``
 ^^^^^^^^^^
-Usage::
+Permanently removes a volume::
 
     rack block-storage volume delete --id <volumeID> [optional flags]
     rack block-storage volume delete --name <volumeName> [optional flags]
     (echo volumeID1 && echo volumeID2) | rack block-storage volume delete --stdin id [optional flags]
 
+**Response**
+
+.. code::
+    
+    $ rack block-storage volume delete --name Response
+    Deleting volume [66dcbe53-1b62-4a15-adc2-e46e78b95f8b]
+
 ``snapshot``
 ~~~~~~~~~~~~
 
-Block Storage Snapshot operations
+Block storage snapshot commands use this syntax::
+
+    rack block-storage snapshot <actions> [optional flags]
 
 ``list``
 ^^^^^^^^
-Usage::
+Retrieves a list of snapshots::
 
     rack block-storage snapshot list [optional flags]
 
+**Response**
+
+.. code::
+
+    $ rack block-storage snapshot list
+    ID				                          	Name	Size	Status		VolumeID				                      VolumeType	SnapshotID	Bootable	Attachments
```

Comment:
This is fine except for the `Attachments` header. That was removed from the `list` output recently due to formatting issues. So you  can leave everything as is except delete the `Attachments` header.